api/src/test/java/org/openmrs/module/dataintegrity/db/impl/DataintegrityDAOImplTest.java

```diff
@@ -1,80 +0,0 @@
-package org.openmrs.module.dataintegrity.db.impl;
-
-import org.openmrs.module.dataintegrity.db.DataintegrityResult;
-import org.openmrs.module.dataintegrity.db.DataintegrityRule;
-import org.hibernate.Criteria;
-import org.hibernate.SQLQuery;
-import org.hibernate.SessionFactory;
```

Comment:
@dkayiwa Saw no need to test the DAO since the tests at the service level cover this